install.sh

```diff
@@ -16,6 +16,7 @@ CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
 CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 CURL="${CURL:-curl}"
 
+# EARLY_PACKAGES cannot depend on crew_profile_base
```

Comment:
Isn't git considered an early package?

Reply:
Yes, but only git's bash completion depends on crew-profile-base, not git itself.